cypress/support/commands.js

```diff
@@ -32,10 +32,29 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+Cypress.Commands.add('register', (email, username, password) => {
+  cy.request('POST', '/users', {
+    email: email,
+    username: username,
+    password: password
+  });
+});
+
+Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
   cy.request('POST', '/users', {
     email,
     username,
     password
+  }).then(response => {
+    const user = {
+      id: response.body.user.id,
+      username: response.body.user.username,
+      email: response.body.user.email,
+      bio: response.body.user.bio,
+      image: response.body.user.image,
+      token: response.body.user.token
+    };
+    window.localStorage.setItem('user', JSON.stringify(user));
```

Comment:
Avoid using 'window' in Cypress commands. Use the 'cy.window()' command instead.